register_service/utils.py

```diff
@@ -0,0 +1,9 @@
+
+import random
+
+
+def short_id(length):
```

Comment:
The DB can accomodate UUIDv4 in their usual dashed notation (and the URLs handle that as well). I changed it to generate these because they look nicer and aren't quite as long. SMS verification isn't clear yet, but we'll probably end up requiring the user to type some ID into a webpage where this kind of ID is helpful (clearly distinguishable character set, not overly long).